tests/components/geonetnz_quakes/test_sensor.py

```diff
@@ -38,27 +34,34 @@
         (38.0, -3.0),
         locality="Locality 1",
         attribution="Attribution 1",
-        time=datetime.datetime(2018, 9, 22, 8, 0, tzinfo=datetime.timezone.utc),
+        time=datetime.datetime(2018, 9, 22, 8, 0,
+                               tzinfo=datetime.timezone.utc),
         magnitude=5.7,
         mmi=5,
         depth=10.5,
         quality="best",
     )
-    mock_entry_2 = _generate_mock_feed_entry(
-        "2345", "Title 2", 20.5, (38.1, -3.1), magnitude=4.6
-    )
-    mock_entry_3 = _generate_mock_feed_entry(
-        "3456", "Title 3", 25.5, (38.2, -3.2), locality="Locality 3"
-    )
-    mock_entry_4 = _generate_mock_feed_entry("4567", "Title 4", 12.5, (38.3, -3.3))
+    mock_entry_2 = _generate_mock_feed_entry("2345",
+                                             "Title 2",
+                                             20.5, (38.1, -3.1),
+                                             magnitude=4.6)
+    mock_entry_3 = _generate_mock_feed_entry("3456",
+                                             "Title 3",
+                                             25.5, (38.2, -3.2),
+                                             locality="Locality 3")
+    mock_entry_4 = _generate_mock_feed_entry("4567", "Title 4", 12.5,
+                                             (38.3, -3.3))
 
     # Patching 'utcnow' to gain more control over the timed update.
     utcnow = dt_util.utcnow()
     with patch("homeassistant.util.dt.utcnow", return_value=utcnow), patch(
-        "aio_geojson_client.feed.GeoJsonFeed.update", new_callable=CoroutineMock
-    ) as mock_feed_update:
-        mock_feed_update.return_value = "OK", [mock_entry_1, mock_entry_2, mock_entry_3]
-        assert await async_setup_component(hass, geonetnz_quakes.DOMAIN, CONFIG)
+            "aio_geojson_client.feed.GeoJsonFeed.update",
+            new_callable=CoroutineMock) as mock_feed_update:
+        mock_feed_update.return_value = "OK", [
+            mock_entry_1, mock_entry_2, mock_entry_3
+        ]
+        assert await async_setup_component(hass, geonetnz_quakes.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)